packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -57,6 +54,7 @@ import { HeaderConnectWalletButton } from '../common/HeaderConnectWalletButton'
 import { Notifications } from '../common/Notifications'
 import { getNetworkName, isNetwork, rpcURLs } from '../../util/networks'
 import { useArbQueryParams } from '../../hooks/useArbQueryParams'
+import { ArbQueryParamProvider } from './ArbQueryParamProvider'
```

Comment:
One final thing: should we maybe keep them together in the same file as they are meant to be used together?

```
import { useArbQueryParams, ArbQueryParamProvider } from '../../hooks/useArbQueryParams'
```